.env.example

```diff
@@ -5,5 +5,5 @@ BOT_TOKEN=yourbottoken
 # Set this to be the url given by ngrok
 BOT_DOMAIN=https://xxxx-xxx-xxx-xx-x.ngrok-free.app
 BOT_PATH=/webhook
-DATABASE_URL=postgres://pg.neon.tech:5432/neondb
-DATABASE_SSL_ENABLED=true
\ No newline at end of file
+DATABASE_URL=postgresql://root:root@localhost:5432/app
+DATABASE_SSL_ENABLED=false
```

Comment:
Erm can we not? I don't really want to oblige the developer to figure out docker etc.

Reply:
Should I just remove the docker compose file and everything?

Reply:
you can keep it there, but perhaps remove the npm run commands and reference to localhost

Reply:
ok!

Reply:
done